packages/engine/src/avatar/components/AnimationComponent.ts

```diff
@@ -73,7 +72,6 @@ export const useLoadAnimationFromBatchGLTF = (urls: string[], keepEntities = fal
 
 export const useLoadAnimationFromGLTF = (url: string, keepEntity = false) => {
   const assetEntity = useHookstate(UndefinedEntity)
-  const simEntity = EntityLayerState.useLinkedEntity(assetEntity.value, 'simulation' as LayerID)
```

Comment:
This logic will break again once we remove having two concurrent component reactors for both simulation and authoring layer gltf component entities